File: src/IsbInterop/Tasks/Proxies/TaskInfo.vb
Namespace Tasks.Proxies

  ''' <summary>
  ''' Обертка над ITaskInfo.
  ''' </summary>
  Friend Class TaskInfo
    Inherits CustomWorkInfo
    Implements ITaskInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwITaskInfo">COM-объект ITaskInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwITaskInfo As Object, scope As IScope)
      MyBase.New(rcwITaskInfo, scope)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop/Tasks/Proxies/TaskInfo.cs
namespace IsbInterop.Tasks.Proxies
{
  /// <summary>
  /// Обертка над ITaskInfo.
  /// </summary>
  internal class TaskInfo : CustomWorkInfo, ITaskInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwITaskInfo">COM-объект ITaskInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public TaskInfo(object rcwITaskInfo, IScope scope) : base(rcwITaskInfo, scope) { }
  }
}
